test/res.sendFile.js

```diff
@@ -615,7 +615,7 @@ describe('res', function(){
         .get('/')
         .end(function(err, res){
           res.text.should.equal('<p>{{user.name}}</p>');
-          res.headers.should.have.property('content-type', 'text/html; charset=UTF-8');
+          should(res.headers).have.property('content-type', 'text/html; charset=UTF-8');
```

Comment:
I'm not familiar with `supertest`, I was just merely implementing a fix that another `should` user had used (for the `querystring` changes in node v6.0.0).